etc/helm/pachyderm/templates/proxy/console-proxy-backend-service.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
   name: console-proxy-backend
   namespace: {{ .Release.Namespace }}
 spec:
+  publishNotReadyAddresses: true
```

Comment:
Do we need to add this to the pachd service as well?

Reply:
That was added in #9227 it seems.